Readme.md

```diff
@@ -32,154 +32,175 @@ We've implemented some version of this framework at several [clients](https://qx
+  `python -m pytest -k table`  
+  This will trigger tests that match the pattern, such as `test_example_table.py`.
+
+- **`--slack_flag`**  
+  *Posts pytest reports on the Slack channel.*  
+  **Example:**  
+  `python -m pytest --slack_flag Y -v > log/pytest_report.log`  
+  This will send the pytest-report on configued slack channel at the end of test run.
+
+- **`-n`**  
+  *Runs tests in parallel.*  
+  **Example:**  
+  `python -m pytest -n 3 -v`  
+  This will run three tests in parallel.
+
+- **`--tesults`**  
+  *Reports test results to Tesults.*  
+  **Example:**  
+  `python -m pytest tests/test_example_form.py --tesults Y`
+
+- **`--interactive_mode_flag`**  
+  *Runs the tests interactively.*  
+  **Example:**  
+  `python -m pytest tests/test_example_form.py --interactive_mode_flag Y`  
+  This option allows the user to pick the desired configuration to run the test from a menu displayed.
+
+  **Note:** If you wish to run the test in interactive mode on Git Bash for Windows, set your bash alias by adding the following command to `.bashrc`:  
+  `alias python='winpty python.exe'`
+
+- **`--summary`**  
+  *Summarizes the pytest results in an HTML report.*  
+  **Example:**  
+  `python -m pytest -k example_table --summary y`  
+  **Note:** You need to provide your `OPENAI_API_KEY` using `export OPENAI_API_KEY=<your-key>`.
+
+### b) Specific Commands
+
+- **Standalone Test**  
+	`python -m pytest tests/test_example_form.py`
+
+- **Run Against Specific Browser**  
+	`python -m pytest tests/test_example_form.py --browser Chrome`
+
+- **API Test**  
+	`python -m pytest tests/test_api_example.py`  
+	**Note:** Ensure the sample `cars-api` is available at `qxf2/cars-api` repository before running the API test.
+
+- **Mobile Test Run on Browserstack/Sauce Labs**  
+	`python -m pytest tests/test_mobile_bitcoin_price --mobile_os_version <android version> --device_name <simulator> --app_path <.apk location on local> --remote_flag Y`  
+	**Note:** For running tests on Browserstack/Sauce Labs, update the Browser_Plaform, Username and AccessKey in `.env.remote` from your Browserstack/Sauce Labs account. Refer our wiki page for more details: [Integrate our Python Selenium automation framework with Cloud Services ](https://github.com/qxf2/qxf2-page-object-model/wiki/Integration-with-Cloud-Services)
+
+- **Run Test along with Tesults**  
+	`python -m pytest tests/test_example_form.py --tesults Y`  
+	**Note:** For running the test along with Tesults, update the .env file with target_token and run the above command. Refer env_conf file for configuration.
+		
+- **Run Tests along with ReportPortal**  
+	`python -m pytest -k example --reportportal`  
+	**Note:** For running the test along with ReportPortal, update the .env file with report portal credential details and run the above command. Refer our wiki page for more details: [Integration with ReportPortal](https://github.com/qxf2/qxf2-page-object-model/wiki/Integration-with-ReportPortal) 
```

Comment:
In this line "update the .env file with report portal credential details " Can you make it ReportPortal.

In the corresponding wiki, can you rephrase slightly the first paragraph.
"Report Portal services are mainly used for analyzing test results. We integrate report analysis into our framework, which collects and stores test log information and screenshot images in the Report Portal."

In general, in the wiki, wherever the word reportportal is referred can you make it "ReportPorta" for consistency.